webf/lib/src/dom/element.dart

```diff
@@ -1002,6 +1002,11 @@ abstract class Element extends Node with ElementBase, ElementEventMixin, Element
 
   @mustCallSuper
   void setAttribute(String qualifiedName, String value) {
+    if (value.startsWith(';')) value = value.substring(1);
```

Comment:
What's the purpos of this line?

Reply:
UICommand pass some attr values start with a semicolon, it seems  it can be ignore handle here .

Reply:
It should't handled here, it's an base road for setAttribute API. Please identify the source of property and value which contains the semicolon and fixed for that property.